### PR TITLE
adds DSM consumption operator to the service list

### PR DESCRIPTION
**Pull request by RenilRaj-BR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +52,7 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | Cloud Native Registry Service - Harbor | ❌ * | ✅ |
 | Kubernetes Ingress Controller Service - Contour | ❌ | ✅ |
 | External DNS Service - ExternalDNS | ❌ | ✅ |
+| DSM Consumption Operator | ❌ | ✅ |
```
Comment (fstrudel) [inline README.md:55]: Your service ID suggests you're releasing a fling `dsm.fling.vsphere.vmware.com`
We expect you will release a supported service? like `dsm.vsphere.vmware.com`, is that correct?

Comment (RenilRaj-BR) [reply README.md:55]: I have removed the `fling` ones

**Review by jswolf059 — APPROVED** [review]
Looks good to me. @fstrudel Are you good with it?

File: README.md [hunk 2]
```diff
@@ -236,16 +239,40 @@ NSX Management Proxy Sample `values.yaml`
 
 - Download latest version: [values for v0.1.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=nsx-management-proxy/v0.1.1/nsx-management-proxy-data-values.yml). Make sure to fill the property `nsxManagers` with your NSX Manager IP(s).
 
+## DSM Consumption Operator
```
Comment (fstrudel) [inline README.md:242]: We should spell out what DSM means.

Comment (RenilRaj-BR) [reply README.md]: done

**Review by fstrudel — APPROVED** [review]

**Pull request merged**